synapse/res/templates/password_reset_confirmation.html

```diff
@@ -1,5 +1,8 @@
-<html>
-<head></head>
+<html lang="en">
+<head>
```

Comment:
Should this have a title? (Not sure if this is intended for use in emails)

The same for `password_reset_failure` and `password_reset_success` too.